posts/2024-02-27_tmc_cran/tmc_cran.qmd

```diff
@@ -0,0 +1,66 @@
+---
+title: "teal.modules.clinical v0.9.0 is now on CRAN!"
+author:
+  - name: Leena Khatri
+description: ""
+date: "2024-02-27"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [community, NEST]
+# feel free to change the image
+image: "teal.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-02-27_tmc_cran"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+![](teal.png){width="184"}
+
+## Introduction
+
+I am thrilled to share the exciting news of the release of `teal.modules.clinical` 0.9.0 on CRAN.
+This significant achievement marks a major milestone for the NEST team in our open-source efforts, that will make a profound impact on the entire open-source community.
+
+## Accelerating clinical insights
+
+Designed to enable faster insights generation, the `teal.modules.clinical` package contains a set of standard teal modules to be used with CDISC data in order to generate many of the common analysis displays used in clinical trials.
```

Comment:
Could you always link to the package when mentioning it?
e.g: 
[{falcon}](https://pharmaverse.github.io/falcon/)